lib/modules/minifyUrls.es6

```diff
@@ -112,9 +110,13 @@ export default function minifyUrls(tree, options, moduleOptions) {
         STORED_URL_BASE = urlBase;
     }
 
-    tree.match({ tag: REGEXP_TAGS_HAVE_URI_VSLUES_FOR_ATTRIBUTES }, node => {
+    tree.walk(node => {
         if (!node.attrs) return node;
 
+        if (!node.tag) return node;
+
+        if (!tagsHaveUriValuesForAttributes.has(node.tag)) return node;
```

Comment:
As I mentioned before, `tree.match` is a little bit slower than `tree.walk`. Use `tree.walk` for simple matching.